providers/pushwoosh/src/lib/pushwoosh.provider.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  IPushOptions,
+  IPushProvider,
+} from '@novu/stateless';
+
+import { Pushwoosh } from 'web-push-notifications';
```

Comment:
A small question @CursedRock17 is there a particular reason why are we using the client library here?
I've came across a node client https://www.npmjs.com/package/pushwoosh-client should we perhaps switch to it.